### PR TITLE
[MM-15191] Migrate Webhook.AnalyticsIncomingCount to Sync

### PR DESCRIPTION
**Pull request by tengis617**
#### Summary
Migrate Webhook.AnalyticsIncomingCount to Sync

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/10645

### CODE REVIEW

File: store/sqlstore/webhook_store.go [hunk 1]
```diff
@@ -331,26 +331,25 @@ func (s SqlWebhookStore) UpdateOutgoing(hook *model.OutgoingWebhook) store.Store
...
 }
 
-func (s SqlWebhookStore) AnalyticsIncomingCount(teamId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		query :=
-			`SELECT 
+func (s SqlWebhookStore) AnalyticsIncomingCount(teamId string) (int64, *model.AppError) {
+	query :=
+		`SELECT 
 			    COUNT(*)
```
Comment (jespino) [inline store/sqlstore/webhook_store.go]: nit: I would unindent this lines one level, to keep it consistent with the previous implementation.

Comment (tengis617) [reply store/sqlstore/webhook_store.go]: @jespino is it correct now? Changed my editor indentation settings to tabs .

File: store/sqlstore/webhook_store.go [hunk 2]
```diff
@@ -331,26 +331,25 @@ func (s SqlWebhookStore) UpdateOutgoing(hook *model.OutgoingWebhook) store.Store
...
-			WHERE
-                DeleteAt = 0`
+func (s SqlWebhookStore) AnalyticsIncomingCount(teamId string) (int64, *model.AppError) {
+	query :=
+		`SELECT 
+			COUNT(*)
+		FROM
+			IncomingWebhooks
+		WHERE
+	DeleteAt = 0`
```
Comment (jespino) [inline store/sqlstore/webhook_store.go]: I would add here 2 extra tabs

Comment (tengis617) [reply store/sqlstore/webhook_store.go]: @jespino done!

**Pull request merged**